### DISCUSSION TITLE
Welcome to zxing-cpp Discussions!

### DISCUSSION THREAD

**Discussion by axxel** [zxing-cpp/zxing-cpp#389] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by axxel** [comment]
My `(community feedback requested)` issues would probably have been better introduced as discussions. One of the unexpected surprises I got from now 'owning' this repository ;).

**Comment by oehhar** [comment]
Harald Oehlmann (oehhar) has no idea about the code or its usage. But gives (often unrelated) comments about Bar Code standards and usage. Thank you, that I may serve sometimes and appogize for all unrelated stuff. You guys rock ! Harald

**Comment by duanqn** [comment]
Hi, I'm Qingnan Duan. I'm a software engineer. Using ZXing-cpp is now part of my job and I'm also interested in making it better (recognition accuracy. performance, crash fix, etc.).
